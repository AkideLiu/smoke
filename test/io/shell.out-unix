global/runs the command with a custom shell
  succeeded
global/runs the filter with a custom shell
  succeeded
local/use the default shell
  succeeded
local/pipe STDIN to the script
  succeeded
local/use custom shell flags
  succeeded
local/use a custom shell
  succeeded
local/pass args to a shell command
  succeeded
local/pass args to a shell command with a custom shell
  succeeded
local/use a custom shell with an absolute path
  succeeded
local/use a shell that doesn't exist
  The executable "non_existent_application" could not be found.
local/use a shell that isn't executable
  The file at "./fixtures/non_executable_application" is not executable.

11 tests, 2 failures
