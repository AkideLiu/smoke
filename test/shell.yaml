tests:
  - name: shell
    args:
      - fixtures/shell
    stdout:
      file: io/shell.out
