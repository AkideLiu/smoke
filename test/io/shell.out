write to STDERR with the default shell
  succeeded
write to STDERR with custom shell flags
  succeeded
write to STDERR with a custom shell
  succeeded
write the args to STDERR
  succeeded
write the args to STDERR with a custom shell
  succeeded
write STDIN to STDERR
  succeeded

6 tests, 0 failures
